providers/apache/hive/src/airflow/providers/apache/hive/hooks/hive.py

```diff
@@ -190,7 +194,11 @@ def _prepare_cli_cmd(self) -> list[Any]:
                 if self.high_availability:
                     if not jdbc_url.endswith(";"):
                         jdbc_url += ";"
-                    jdbc_url += "serviceDiscoveryMode=zooKeeper;ssl=true;zooKeeperNamespace=hiveserver2"
+                    ssl = conn.extra_dejson.get("ssl", "true")
```

Comment:
```suggestion
                    ssl = conn.extra_dejson.get("ssl", True)
```
? its defined BooleanField at line: 145